src/desktop/src/libs/helpers.js

```diff
@@ -13,6 +13,68 @@ export const capitalize = (input) => {
  * @param {number} length - The target length
  * @param {string} Trimmed string
  */
-export const shorten = (input, length) => {
+const shorten = (input, length) => {
     return typeof input === 'string' && input.length > length ? input.substr(0, length - 1) + '…' : input;
 };
+
+/**
+ * Tryte trit mapping
+ */
+const trytesTrits = [
+    [0, 0, 0],
+    [1, 0, 0],
+    [-1, 1, 0],
+    [0, 1, 0],
+    [1, 1, 0],
+    [-1, -1, 1],
+    [0, -1, 1],
+    [1, -1, 1],
+    [-1, 0, 1],
+    [0, 0, 1],
+    [1, 0, 1],
+    [-1, 1, 1],
+    [0, 1, 1],
+    [1, 1, 1],
+    [-1, -1, -1],
+    [0, -1, -1],
+    [1, -1, -1],
+    [-1, 0, -1],
+    [0, 0, -1],
+    [1, 0, -1],
+    [-1, 1, -1],
+    [0, 1, -1],
+    [1, 1, -1],
+    [-1, -1, 0],
+    [0, -1, 0],
+    [1, -1, 0],
+    [-1, 0, 0],
+];
+
+/**
+ * Convert trit to an ASCII character
+ * @param {trit} trit - raw Trit input
+ */
+const byteToChar = (trit) => {
+    return '9ABCDEFGHIJKLMNOPQRSTUVWXYZ'.charAt(trit % 27);
+};
+
+/**
+ * Convert single character string to trit array
+ * @param {string} char - Input character
+ * @returns {array} Output trit array
+ */
+const charToByte = (char) => {
+    return '9ABCDEFGHIJKLMNOPQRSTUVWXYZ'.indexOf(char);
+};
+
+const byteToTrit = (byte) => {
+    return trytesTrits[byte % 27];
+};
+
+module.exports = {
```

Comment:
Was there an issue with exporting the constant itself?

Reply:
`libs/helpers` is now used also on the main process and that is not going through Babel, so no ES6, here :/